### PR TITLE
Allow transformation functions for snapshots

### PR DESCRIPTION
**Pull request by emcastillo**
Follows #7

### CODE REVIEW

File: pytorch_pfn_extras/training/extensions/_snapshot.py [hunk 1]
```diff
@@ -142,6 +142,12 @@ def snapshot_object(target, filename, savefun=None, **kwargs):
             Automatic loading only works when the filename is a string.
         saver_rank (int): If defined, the snapshot will be taken by only one
             rank when running in distributed mode and restored by all.
+        transform_models (func): If defined, function to apply to a model
```
Comment (asi1024) [inline pytorch_pfn_extras/training/extensions/_snapshot.py:145]: We write `callable` conventionally in such a situation.
```suggestion
        transform_models (callable): If defined, function to apply to a model
```

**Comment by emcastillo** [comment]
PTAL!

File: pytorch_pfn_extras/training/extensions/_snapshot.py (hunk 1 above)
Comment (asi1024) [reply pytorch_pfn_extras/training/extensions/_snapshot.py]: Could you fix this part too?
https://github.com/pfnet/pytorch-pfn-extras/pull/16/files#diff-21bebc7a5bc35433c970655418a04770R227-R232

**Comment by emcastillo** [comment]
Should be fixed now

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**